pincer/objects/guild/guild.py

```diff
@@ -486,6 +496,257 @@ async def kick(self, member_id: int):
         """
         await self._http.delete(f"guilds/{self.id}/members/{member_id}")
 
+    async def get_roles(self) -> AsyncGenerator[Role, None]:
+        """|coro|
+        Fetches all the roles in the guild.
+
+        Returns
+        -------
+        AsyncGenerator[:class:`~pincer.objects.guild.role.Role`, :data:`None`]
```

Comment:
```
:class:`None`
```

Reply:
`None` is a singleton of the class `NoneType` so it should be data. Also the doc hyperlinks only work if its data.